tests/beam_tests.ts

```diff
@@ -771,7 +772,7 @@ function complexWithAnnotation(options: TestOptions): void {
   ];
 
   const font = {
-    family: Font.SERIF,
+    family: Tables.lookupMetric('Annotation.fontFamily'),
```

Comment:
Is Tables a public class? I always thought it was for internal calculations.

I think the files under `tests/*_tests.ts` should be representative of how a developer would use VexFlow. So if a developer can't easily access Tables, we shouldn't import it and should not use it here.

Reply:
@ronyeh lookupMetric should be public. I suggest to leave the solution out of this PR. Perhpas tables is not the right place or we need and access function from Vex.Flow.